Tasks/GooglePlayReleaseV4/task.json

```diff
@@ -237,6 +237,14 @@
             "helpMarkDown": "Select this option to send changes for review in GooglePlay Console. If changes are already sent for review automatically, you shouldn't select this option. [More info](https://developers.google.com/android-publisher/api-ref/rest/v3/edits/commit#query-parameters).",
             "groupName": "advanced"
         },
```

Comment:
Please bump relevant task version - see [steps](https://github.com/microsoft/google-play-vsts-extension/tree/master/docs/taskversionbumping.md) for more details

Reply:
Bumped the minor version to 204